src/operator/c_lapack_api.h

```diff
@@ -242,6 +249,20 @@ inline void flip(int m, int n, DType *b, int ldb, DType *a, int lda) {
   #define MXNET_LAPACK_sgetrf LAPACKE_sgetrf
   #define MXNET_LAPACK_dgetrf LAPACKE_dgetrf
 
+  #define MXNET_LAPACK_CWRAP_GESVD(prefix, dtype) \
+  inline int MXNET_LAPACK_##prefix##gesvd(int matrix_layout, int m, int n, dtype* ut, \
```

Comment:
Not sure I understand this case. Maybe add a comment?

Reply:
The LAPACK_gesvd function interface differs in signature from the MXNET_LAPACK-signature and have to be wrapped (as is stated [here](https://github.com/apache/incubator-mxnet/blob/67191c4df9ba363605c59b332c1a8096573309e0/src/operator/c_lapack_api.h#L201)). So this is basically a wrapper of LAPACK_gesvd.

I added some comments about how to use the LAPACK_gesvd. Its official document can be found [here](https://software.intel.com/en-us/mkl-developer-reference-c-gesvd).